mne/io/brainvision/brainvision.py

```diff
@@ -252,7 +252,14 @@ def _check_mrk_version(header):
 _fmt_dict = dict(INT_16='short', INT_32='int', IEEE_FLOAT_32='single')
 _fmt_byte_dict = dict(short=2, int=4, single=4)
 _fmt_dtype_dict = dict(short='<i2', int='<i4', single='<f4')
-_unit_dict = {'V': 1., u'µV': 1e-6, 'uV': 1e-6}
+_unit_dict = {'V': 1.,
+              u'µV': 1e-6,
+              'uV': 1e-6,
+              'C': 1,
+              u'µS': 1e-6,
+              u'uS': 1e-6,
+              u'ARU': 1,
+              'S': 1}
```

Comment:
@Pablo-Arias please add a comment on these lines like what you mention on GitHub:

S = Siemens; 
V = Volt; 
C = celsius 
and
ARU is the unity for the breathing data